corpus/templates/emails/impulse/announcement.html

```diff
@@ -6,7 +6,9 @@
 
 {% block content %}
     <h1>Announcement - Impulse</h1>
-    <p>{{ announcement.content }}</p>
+    <div style="font-size: 1.2em;">
+        {{ announcement.content | safe }}
```

Comment:
As mentioned above

Reply:
You should also update this in the HTML page that you are displaying to participants. 

Also, could you make the same change in the corresponding places in Embedathon, i.e. in the mail template and the participant dashboard?

Reply:
Yep, replaced in all templates.